pkg/daemon/ceph/client/pool.go

```diff
@@ -529,7 +529,15 @@ func extractPoolDetails(rule ruleSpec) (string, string) {
 			failureDomain = step.Type
 		}
 		if step.ItemName != "" {
```

Comment:
How about adding a unit test for this? For example, could add to the existing test: `TestExtractPoolDetails`